nautobot/extras/models/customfields.py

```diff
@@ -131,13 +131,13 @@ class CustomField(BaseModel):
     weight = models.PositiveSmallIntegerField(
         default=100, help_text="Fields with higher weights appear lower in a form."
     )
-    validation_minimum = models.PositiveIntegerField(
+    validation_minimum = models.PositiveBigIntegerField(
```

Comment:
I'm actually not sure why these need to be *Positive* fields in the first place. Thoughts on changing them to simply `BigIntegerField` so as to allow the option of including negative numbers in the validation range as well?

Reply:
It makes sense to keep them integers instead of positive integers. I will change them to `BigIntegerField` instead

Reply:
https://github.com/Joezeppe/nautobot/commit/96d97e5c9dd6d4211ba5a258e2a6846e672d8cbc#diff-8d1c47fce517d61985f7f5a7ae57a557cfd35f185221ff4fd5a73bececcad429R227